NitroxClient/GameLogic/TimeManager.cs

```diff
@@ -37,11 +38,42 @@ public double CurrentTime
         }
     }
 
+    /// <summary>
+    ///     Real deltaTime between two updates of local time through <see cref="DayNightCycle_Update_Patch"/>
+    /// </summary>
+    /// <remarks>
+    /// <para>
+    ///     Replaces <see cref="Time.deltaTime"/> because it is capped by <see cref="Time.maximumDeltaTime"/>
+    ///     and may not reflect the real time which has passed between two frames once it's higher than the said maximum
+    ///     [<see href="https://docs.unity3d.com/ScriptReference/Time-maximumDeltaTime.html"/>].
+    /// </para>
+    /// <para>
+    ///     This value is set to <c>0</c> when a time skip occurs to avoid undesired behaviours
+    ///     (e.g. consuming a day worth of energy just when you skipped 24 in-game hours)
+    /// </para>
+    /// </remarks>
```

Comment:
```suggestion
    /// <summary>
    ///     Real deltaTime between two updates of local time through <see cref="DayNightCycle_Update_Patch"/>
    /// </summary>
    /// <remarks>
    /// <para>
    ///     Replaces <see cref="Time.deltaTime"/> because it is capped by <see cref="Time.maximumDeltaTime"/>
    ///     and may not reflect the real time which has passed between two frames once it's higher than the said maximum
    ///     <br/>See: <a href="https://docs.unity3d.com/ScriptReference/Time-maximumDeltaTime.html">Time.maximumDeltaTime</a>
    /// </para>
    /// <para>
    ///     This value is set to <c>0</c> when a time skip occurs to avoid undesired behaviours
    ///     (e.g. consuming a day worth of energy just when you skipped 24 in-game hours)
    /// </para>
    /// </remarks>
```

![image](https://github.com/SubnauticaNitrox/Nitrox/assets/1107063/3ca317c2-4107-47e9-8b5a-fa1f2c8bdee2)